q2_feature_table/_core_features/core_features_assets/index.html

```diff
@@ -0,0 +1,40 @@
+{% extends 'base.html' %}
+
+{% block content %}
+
+<h1>Core features</h1>
+
+<div class='row'>
+  <div class='col-md-6'>
+    <div class='row'>
+      <div class='col-md-12'>
+        <h3>Feature table summary</h3>
+        Sample count: {{num_samples}}<br>
+        Feature count: {{num_features}}
+      </div>
+    </div>
+    <div class='row'>
+      <div class='col-md-12'>
+        <img src='core-feature-counts.svg'>
+      </div>
+    </div>
+  </div>
+  <div class='col-md-6'>
+    <div class='row'>
+      <div class='col-md-12'>
+        <h3>Terminology key</h3>
+        <i>Fraction of samples</i>: The fraction of the total number of samples that a feature must be observed more than zero times in for that feature to be considered "core".<br>
+        <i>Feature count</i>: The number of "core" features at a given fraction of samples.<br>
+        <i>Fraction of features</i>: The fraction of the total number of features that are "core" features at a given fraction of samples.<br>
+        <i>Feature list</i>: The list of "core" feature ids and their percentile frequencies at a given fraction of samples.
```

Comment:
Discussed offline with @ebolyen and we decided that `across` doesn't make sense, and to drop the phrase "at a given fraction of samples" since it could be confused with computing the percentile frequencies using only those samples where the "core" feature is observed. We'll also add the word "total" to make this clearer.